README.md

```diff
@@ -8,15 +8,16 @@ By default it does not overwrite any files that may exist with the pre-stablishe
 
 ## Requires
 
-- [Laravel 5](https://laravel.com)
+- [Laravel](https://laravel.com)
 - [rappasoft/laravel-5-boilerplate](https://www.github.com/rappasoft/laravel-5-boilerplate/)
 
-It has been tested with Laravel 5.7.
+It has been tested with Laravel 6.0.
 
 ## Install
 
 ```
 composer require pqrs/l5b-crud
+composer require laravel/helpers
```

Comment:
shouldn't this be in the composer as a dependancy ?

Reply:
certainly would make sense to.. but didn't know the authors comfort level with changing the dependancies.